### PR TITLE
Extract CSS to file in production builds

### PR DESCRIPTION
**Pull request by ngotchac**
Export all the CSS to a proper file if built in production environment.
This prevents a flash of unstyled content on first load.

NB : this is not possible in dev mode since hot reloading doesn't play nice with external CSS files

### CODE REVIEW

**Comment by jacogr** [comment]
Excuse my ignorance - but if gets added automagically to the index.html file

**Comment by ngotchac** [comment]
@jacogr Yep it does, as the JS files actually, thanks to `HtmlWebpackPlugin`

**Comment by jacogr** [comment]
Cool. Will just build with `--no-default-features --features ui` in the morning to give it a run-through

**Comment by jacogr** [comment]
It seems to put the styles for everything in one, e.g. 

from `tokenreg.html` - `<link href="/styles.43f703cc4465bd69701b.css" rel="stylesheet"></head>`
from `index.html` - `<link href="/styles.43f703cc4465bd69701b.css" rel="stylesheet">`

There needs to be a split between the UI, dapps and between the dapps.

**Comment by ngotchac** [comment]
Ah that's right, I'll look into that

**Comment by ngotchac** [comment]
Ok this should be fixed now. Each dapp has it's own CSS file. Plus some files are now in subfolders: `assets` for images, `styles` for CSS, `fonts` for... fonts.

Also fixed `DAPPS_URL` not being used everywhere ; now we can test the builds properly

File: js/webpack/app.js [hunk 1]
```diff
@@ -82,28 +83,51 @@ module.exports = {
         ]
       },
 
+      // // Don't extract CSS for Dapps
```
Comment (jacogr) [inline js/webpack/app.js]: Still want this in?

Comment (ngotchac) [inline js/webpack/app.js]: Fixed

**Comment by jacogr** [comment]
Merge in master to just run the checks.

**Pull request merged**